pyblish_qml/control.py

```diff
@@ -353,29 +353,24 @@ def iterator(self, plugins, context):
                     signals.pop(order).emit()
 
             if not self.data["state"]["is_running"]:
-                return StopIteration("Stopped")
+                return
```

Comment:
I think we're missing a yield here, to preserve the message "Stopped".

```py
yield "Stopped"
return
```

Reply:
By removing the StopIteration exception, I considered that all messages that needed to be transferred were error messages which could be embedded into other type of Exception (e.g. RuntimeError), but this would indeed not work in this case as we wouldn't want to raise a RuntimeError just to get the "Stopped" message.

Yielding "Stopping" as you suggest would imply a check on the "on_next" callback to verify if "result" is a string, which is what I wanted to prevent.

So I think I'm gonna revert to simply apply the logic recommended in this comment instead:
https://github.com/pyblish/pyblish-qml/issues/353#issuecomment-584159312

Reply:
Done